### PR TITLE
SLING-7975 document ordering of resources

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

File: src/main/jbake/content/documentation/the-sling-engine/sling-api-crud-support.md [hunk 1]
```diff
@@ -150,14 +151,31 @@ Delete the resource /myresource
     resourceResolver.delete(myResource);
     resourceResolver.commit();
 
-## Value Class Support
+## Order a Resource
+
+Delete the resource /myresource
```
Comment (rombert) [inline src/main/jbake/content/documentation/the-sling-engine/sling-api-crud-support.md:156]: I think this is incorrect.

**Review by rombert — COMMENTED** [review]
LGTM, with (probably) a typo. Did not review the resource access security part as I'm not familiar with it.

File: src/main/jbake/content/documentation/the-sling-engine/sling-api-crud-support.md (hunk 1 above)
Comment (kwin) [reply src/main/jbake/content/documentation/the-sling-engine/sling-api-crud-support.md]: Thanks, fixed in https://github.com/apache/sling-site/pull/72/commits/f4d78988c78b11011821ef638bb6254f3f8cc51a

Comment (rombert) [reply src/main/jbake/content/documentation/the-sling-engine/sling-api-crud-support.md]: Thanks.

**Pull request merged**